legal-api/src/legal_api/models/business.py

```diff
@@ -126,6 +126,15 @@ def delivery_address(self):
         return db.session.query(Address).filter(Address.business_id == self.id).\
             filter(Address.address_type == Address.DELIVERY)
 
+    @property
+    def good_standing(self):
+        """Return true if in good standing, otherwise false."""
+        last_ar = self.founding_date
+        if self.last_ar_date:
+            last_ar = self.last_ar_date
```

Comment:
Is this the date of the last annual report?

Reply:
I was under the impression it was, is there something else it can be? The field name seemed pretty explicit